core/src/language/mod.rs

```diff
@@ -184,7 +268,16 @@ pub trait Language {
     ) -> Result<String, RustTypeFormatError>;
 
     /// Implementors can use this function to write a header for typeshared code
-    fn begin_file(&mut self, _w: &mut dyn Write) -> std::io::Result<()> {
+    fn begin_file(&mut self, _w: &mut dyn Write, _parsed_data: &ParsedData) -> std::io::Result<()> {
+        Ok(())
+    }
+
+    /// For generating import statements.
+    fn write_imports(
+        &mut self,
+        _writer: &mut dyn Write,
+        _imports: ScopedCrateTypes<'_>,
+    ) -> std::io::Result<()> {
         Ok(())
     }
```

Comment:
This shouldn't have a default implementation

Reply:
Updated